pkg/codegen/python/python_test.go

```diff
@@ -0,0 +1,40 @@
+package python
+
+import "testing"
+
+var pyNameTests = []struct {
+	input    string
+	expected string
+	legacy   string
+}{
+	{"kubeletConfigKey", "kubelet_config_key", "kubelet_config_key"},
```

Comment:
Mind adding some more cases mentioned in the original bug?

"non_resource_ur_ls": "nonResourceURLs"
"pod_i_ps": "podIPs"
"pod_cid_rs": "podCIDRs"